xml/System.Security.Cryptography.X509Certificates/X500DistinguishedNameBuilder.xml

```diff
@@ -14,8 +14,26 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>This class facilitates building a distinguished name for an X.509 certificate.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+When constructing the SEQUENCE OF Relative Distinguished Names, this builder constructs the SEQUENCE OF in the opposite order which they were added to the builder. For example:
+
+<code>
```

Comment:
Since this is inside a text/markdown section, `<code>` should probably be three backticks?

Reply:
You are correct. And you found a bug in the api-docs-sync tool. 😃

Reply:
Actually, there's no bug in the tool. I just remembered this was manually edited by me, duh.